var/spack/repos/builtin/packages/py-bluepyopt/package.py

```diff
@@ -13,6 +13,7 @@ class PyBluepyopt(PythonPackage):
     url = "https://pypi.io/packages/source/b/bluepyopt/bluepyopt-1.9.27.tar.gz"
 
     # NOTE : while adding new release check pmi_rank.patch compatibility
+    version('1.9.38', sha256='fb1411c6a8fbfac52d36b837225bae882fd6524acfb4d0580189312ef3c1cfcc')
```

Comment:
1.10.38?

Reply:
that's a yikes

does the hash not get checked?

Reply:
good eye @genric; thanks.